### PR TITLE
Feature/#532 editable contact person

### LINKED ISSUE #532: Make editing existing contact persons possible
Editing previously created tags and contact persons should be editable. Clicking the name of the tag or contact person should bring up an editor window similar to the window where the tag/person was created

### PR DESCRIPTION
**Pull request by dotanthanh**
Resolved #532.

### CODE REVIEW

File: src/i18n/fi.json [hunk 1]
```diff
@@ -159,6 +159,8 @@
   "eyeTooltipOpensHours": "tunnin kuluttua. Se ei ole vielä julkisesti nähtävissä.",
   "eyeTooltipOpensDays": "päivän kuluttua. Se ei ole vielä julkisesti nähtävissä.",
   "createContact": "Luo uusi yhteyshenkilö",
+  "editContact": "Muokkaa yhteystietoa",
+  "save": "Tallentaa",
```
Comment (jonnetanninen) [inline src/i18n/fi.json]: Should be: "Tallenna"

File: src/actions/hearingEditor.js [hunk 2]
```diff
@@ -157,6 +158,28 @@ export function addContact(contact, selectedContacts) {
...
+export function saveContact(contact) {
+  return (dispatch, getState) => {
+    const url = `/v1/contact_person/${contact.id}/`;
+    const contactInfo = omit(contact, ['id']);
+    return api
+      .put(getState(), url, contactInfo)
+      .then(checkResponseStatus)
+      .then(response => {
+        if (response.status === 400) {
+          notifyError('Tarkista yhteyshenkilön tiedot.');
```
Comment (jonnetanninen) [inline src/actions/hearingEditor.js]: I think this should be something like "Sinulla ei ole oikeutta muokata yhteyshenkilöä." as this error comes when trying to edit person from a organization I don't have edit rights for

Comment (dotanthanh) [reply src/actions/hearingEditor.js]: Should we use the error message from the response to display instead ?

File: src/i18n/fi.json (hunk 1 above)
Comment (dotanthanh) [reply src/i18n/fi.json]: fixed.

File: src/actions/hearingEditor.js (hunk 2 above)
Comment (dotanthanh) [reply src/actions/hearingEditor.js]: fixed.

**Pull request merged**